tests/interfaces/test_tf.py

```diff
@@ -426,7 +427,7 @@ def circuit(x, y, input_state=np.array([0, 0])):
             qml.RY(y, wires=[0])
             return qml.expval(qml.PauliZ(0))
 
-        circuit = qml.QNode(circuit, qubit_device_2_wires).to_tf()
+        circuit = qml.QNode(circuit, qubit_device_2_wires, interface='tf')
```

Comment:
This change looks good, how come `to_tf()` doesn't work any more though?

Reply:
My bad, both should work equally good. I'll revert his change, thanks!